spec/features/request_a_visit_spec.rb

```diff
@@ -79,7 +79,7 @@
     click_button 'Continue'
 
     within('.date-box-2') do
-      click_button "Change date and time slot"
+      click_button "Change slot 2 date and time slot"
```

Comment:
Well spotted that it didn't make sense. I should have removed the last mention of 'slot'. This has now been corrected to Kaye's instructions.